lang/gcc45/Portfile

```diff
@@ -109,7 +108,7 @@ platform darwin {
 
 configure.dir       ${workpath}/build
 configure.cmd       ${worksrcpath}/configure
-configure.args      --enable-languages=c,c++,objc,obj-c++,fortran,java \
+configure.args      --enable-languages=c,c++,objc,obj-c++,java,fortran \
```

Comment:
why invert the order of fortran and java in this option ?

Reply:
In a previous change I proposed to GCC, I was told that maintaining some modicum of consistency between the GCC ports was important.
Since then, I have strived to do so.
Hence the change.